### DISCUSSION TITLE
article/k8gjtpf8/

### DISCUSSION THREAD

**Discussion by giscus** [ermaozi/ermao.net#6] [Announcements]
# article/k8gjtpf8/

近期的事情还挺多。 当然了，以往的事情也并不少，只是记不得去写。今天闲来无事，翻了翻陌生人的博客，感觉琐事挺有意思的。 也好久没写过关于自己的流水账了。 一 • 被阉掉的大龙 大龙被阉掉这件事，绝对是最值得念叨的。 原本计划等大龙两岁的时候给它留个种。 可是发情后的大龙天天鬼哭狼嚎。每次发情持续一周，休息一周后继续发情。年前一段时间被邻居投诉两次。从网...

https://www.ermao.net/article/k8gjtpf8/

**Comment by namewyf** [comment]
怎么没人
